plugins/HtoZAAnalyzer.cc

```diff
@@ -732,9 +739,16 @@ void HtoZAAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, con
 
     // HT: the two selected leptons - if present - plus all selected jets
     HT = 0;
-    if (lljj.size() > 0)
-        // take the first lljj since it's been resized: it has only one entry
-        HT += lljj[0].lep1_p4.Pt() + lljj[0].lep2_p4.Pt();
+    if (lljj_cmva.size() > 0)
+        // take the first lljj_cmva since it's been resized: it has only one entry
+        HT += lljj_cmva[0].lep1_p4.Pt() + lljj_cmva[0].lep2_p4.Pt();
+    for (unsigned int ijet=0; ijet < jets.size(); ijet++) {
+        HT += jets[ijet].p4.Pt();
+    }
+    
+    if (lljj_deepCSV.size() > 0)
+        // take the first lljj_cmva since it's been resized: it has only one entry
+        HT += lljj_deepCSV[0].lep1_p4.Pt() + lljj_deepCSV[0].lep2_p4.Pt();
     for (unsigned int ijet=0; ijet < jets.size(); ijet++) {
         HT += jets[ijet].p4.Pt();
     }
```

Comment:
This doesn't work, there should be two `HT` variables depending on the btag algo and hence on the jets picked...

Reply:
Ah yes sorry, I missed it. Fixiiing